spec/features/visitors/sign_up_spec.rb

```diff
@@ -53,93 +53,25 @@
     click_button 'Submit'
   end
 
-  # Scenario: Visitor is given option to receive OTP via both email & mobile
-  #   Given I am not signed in
-  #   When I sign up with a valid email address and password and click my confirmation link
-  #   Then I am prompted to choose a method for receiving one-time passwords (OTPs)
-  #   And I see that both mobile and email are selected by default
-  scenario 'visitor sees both email and mobile checked by default' do
-    sign_up_with_and_set_password_for('test@example.com')
-
-    expect(find('#user_second_factor_ids_email')).to be_checked
-    expect(find('#user_second_factor_ids_mobile')).to be_checked
-  end
-
   scenario 'form should have autocomplete turned off' do
     sign_up_with_and_set_password_for('test@example.com')
     expect(find('#edit_user')[:autocomplete]).to eq 'off'
   end
 
-  # Scenario: Visitor can sign up, confirm email, and setup 2FA
-  #   Given I am not signed in
-  #   When I sign up, confirm email, and setup 2FA
-  #   Then I see the dashboard
-  context 'visitor can sign up and confirm a valid email for OTP', email: true do
-    before do
-      sign_up_with_and_set_password_for('test@example.com')
-      check 'Email'
-      uncheck 'Mobile'
-      click_button 'Submit'
-    end
-
-    it 'sends an email with an OTP' do
-      open_email('test@example.com')
-
-      expect(last_email.body).to have_content 'Please enter this secure one-time password:'
-    end
-
-    it 'allows the user to sign in with the emailed OTP' do
-      otp = last_email.body.match(/secure one-time password: (\w*)/)
-
-      fill_in 'Secure one-time password', with: otp[1]
-      click_button 'Submit'
-
-      expect(current_path).to eq dashboard_index_path
-      expect(User.find_by_email('test@example.com').second_factor_confirmed_at).to be_present
-      expect(page).to have_content(successful_account_creation_notice)
-    end
-
-    it 'does not include a link to enter a number again' do
-      expect(page).to_not have_link 'entering it again'
-    end
-
-    it 'does not update mobile_confirmed_at' do
-      otp = last_email.body.match(/secure one-time password: (\w*)/)
-
-      fill_in 'Secure one-time password', with: otp[1]
-      click_button 'Submit'
-
-      expect(User.find_by_email('test@example.com').mobile_confirmed_at).to be_nil
-    end
-
-    it 'disables OTP lockout during account creation' do
-      Devise.max_login_attempts.times do
-        fill_in 'Secure one-time password', with: '12345678'
-        click_button 'Submit'
-      end
-
-      expect(page).to_not have_content t('upaya.titles.account_locked')
-      visit user_two_factor_authentication_path
-      expect(current_path).to eq user_two_factor_authentication_path
-    end
```

Comment:
Could we keep this test please? The one about disabling OTP lockout during account creation.